tests/dummy/app/components/login-form.js

```diff
@@ -12,8 +12,8 @@ export default Component.extend({
         .then(() => {
           this.get('rememberMe') && this.set('session.store.cookieExpirationTime', 60 * 60 * 24 * 14);
         })
-        .catch((reason) => {
-          this.set('errorMessage', reason.error);
+        .catch((response) => {
+          response.clone().json().then((responseBody) => this.set('errorMessage', responseBody.error));
```

Comment:
why do we have to `.clone()` it?

Reply:
We are [reading the response body in the authenticator](https://github.com/simplabs/ember-simple-auth/blob/master/addon/authenticators/oauth2-password-grant.js#L314) already and it can only be read once. We could of course clone in the authenticator before rejecting so you wouldn't have to do that when catching it but that might be wasteful in case you only need the response headers.

Reply:
is this documented somewhere?

Reply:
added a note here: #2141